db/migrate/20190213184045_add_allows_article_edits_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddAllowsArticleEditsToUsers < ActiveRecord::Migration[5.1]
+  def change
+    add_column :users, :allows_article_edits, :boolean, default: false
```

Comment:
I think we actually want this to default to `true`

Reply:
Cool, that makes sense!